### DISCUSSION TITLE
Can I get DarkReader to enable on an iFrame of my choosing?

### DISCUSSION THREAD

**Discussion by CraigRodrigues** [darkreader/darkreader#8777] [Q&A]
Quick question. Was wondering if I had an iframe on a page if I can direct DarkReader to enable on it without having the script be loaded inside that iframe?

**Comment by Gusted** [comment]
That's not possible.

**Reply by Gusted** [reply]
Please see https://github.com/darkreader/darkreader/pull/5085
